### DISCUSSION TITLE
✅  sftp to synology nas (usind scp for file transfer)

### DISCUSSION THREAD

**Discussion by ap2de** [isontheline/pro.webssh.net#499] [Q&A] [answered]
I am using a synology nas (DSM 7.0.1-42218).
I have set up ssh and I am able to control the nas from a windows 10 pc using putty as well as winscp.

winscp shows the following information for sevrer and protocol:
Remote system = Linux *** 4.4.180+ #42218 SMP Mon Oct 18 19:17:56 CST 2021 x86_64 GNU/Linux synology_apollolake_1019+
File transfer protocol= SCP
Cryptography protocol= SSH-2
SSH-Implementation = OpenSSH_8.2
Encryption algorithm= AES-256 SDCTR (AES-NI accelerated)
Compression= No

Using your nice winssh app I receive an error, that the server does not support sftp.
How have i got to configure webssh as to use it with my synology nas?

Thank you for your reply!

**Comment by isontheline** [comment, marked as answer]
Hello @ap2de 👋

You need to configure your NAS in order to support SFTP instead of SCP.

WebSSH doesn't support SCP yet.

Could you try?

**Reply by ap2de** [reply]
> WebSSH doesn't support SCP yet.

Thank you for the feedback.
If you cant do it, i will re configure my synology nas as to support also sftp.

**Reply by isontheline** [reply]
SCP support could be tracked here : https://github.com/isontheline/pro.webssh.net/issues/128
